libraries/redshop/tags/sections/clicktellsms.php

```diff
@@ -0,0 +1,117 @@
+<?php
+/**
+ * @package     RedSHOP.Library
+ * @subpackage  Tags
+ *
+ * @copyright   Copyright (C) 2008 - 2020 redCOMPONENT.com. All rights reserved.
+ * @license     GNU General Public License version 2 or later; see LICENSE
+ */
+
+defined('_JEXEC') or die;
+
+/**
+ * Tags replacer abstract class
+ *
+ * @since __DEPLOY_VERSION__
+ */
+class RedshopTagsSectionsClicktellSms extends RedshopTagsAbstract
+{
+	/**
+	 * @var    array
+	 *
+	 * @since __DEPLOY_VERSION__
+	 */
+	public $orderData = array();
+
+	/**
+	 * @var    integer
+	 *
+	 * @since __DEPLOY_VERSION__
+	 */
+	public $paymentName = '';
+
+	/**
+	 * Init
+	 *
+	 * @return  void
+	 *
+	 * @since __DEPLOY_VERSION__
+	 */
+	public function init()
+	{
```

Comment:
@hoang-tran-redweb please setup PSR-12 on PHPStorm and format code.
Thanks,
H.A

Reply:
@anhredweb I setup PSR-12 and format code, please check